modes/elixir-mode/run-elixir.lisp

```diff
@@ -0,0 +1,86 @@
+(defpackage :lem-elixir-mode.run-elixir
+  (:use :cl :lem :lem-elixir-mode)
+  (:export :*elixir-run-command*
+           :run-elixir))
+
+(in-package :lem-elixir-mode.run-elixir)
+
+(defvar *elixir-run-command* "iex")
+
+(defvar *process* nil)
+
+(define-major-mode run-elixir-mode lem-elixir-mode:elixir-mode
+    (:name "elixir"
+     :keymap *run-elixir-mode-keymap*
+     :syntax-table lem-elixir-mode::*elixir-syntax-table*)
```

Comment:
Wouldn't this be better to export the symbol?
The same goes for `*elixir-mode-keymap*`.

Reply:
Done, this was because I just copy and paste from the python mode, should it also be changed there?

Reply:
Yes, that was an oversight.